README.md

```diff
@@ -131,7 +131,13 @@ The following will train a _nerfacto_ model, our recommended model for real worl
 # Download some test data:
 ns-download-data nerfstudio --capture-name=poster
 # Train model
-ns-train nerfacto --data data/nerfstudio/poster
+ns-train nerfacto --data data/nerfstudio/poster 
+```
+The last command-line can be also replaced by referring explicitly to the transformation file:
+
+```bash
+ns-train nerfacto --data data/nerfstudio/poster/transforms.json
+
```

Comment:
Let's avoid adding this to the readme, I think the readme should be as simple as possible. This info should exist in the config help text ie. - https://github.com/nerfstudio-project/nerfstudio/blob/1a70665e76bfbc71ef193cc6d3429efa582ddaff/nerfstudio/data/dataparsers/nerfstudio_dataparser.py#L50

Reply:
@tancik ,what am I doing wrong with the Readme ? I reverted the changes and still dont pass the doc builds.
locally I pass on my station.

Reply:
It seems to be a bug with readthedocs. This just started happening to other PRs. Ignore for now.

Reply:
Once you add the `.json` optional comments to the dataparsers, I think this PR will be ready to go. I will need to force merge it on my end.